notimetowait2.md

```diff
@@ -79,6 +79,10 @@ The schedule for the 2016 No Time to Wait may be viewed [here](https://docs.goog
 
 The schedule of the event is under development, if you have an idea for the agenda item or presentation, please feel welcome to contact [info@mediaarea.net](mailto:info@mediaarea.net).
 
+## Code of Conduct
+
+This symposium is governed by the [Contributor Covenant version 1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct/). All coordinators, contributors, and participants agree to abide by its terms. To report violations, send an email to info@mediaarea.net or contact a symposium coordinator.
```

Comment:
I suggest conduct@mediaarea.net (name taken from [Django CoC](https://www.djangoproject.com/conduct/) as info@ is used for a lot of other things and it would be more complex to change the recipient in case the person behind info@ is part of the issue. Additionally, this would permit to forward to more than 1 person specifically in charge of the code of conduct.

` send an email to [conduct@mediaarea.net](mailto:conduct@mediaarea.net?subject=Code%20Of%20Conduct)`

Reply:
updated